src/components/navbar/navbar.module.scss

```diff
@@ -319,4 +341,13 @@ header {
       }
     }
   }
+  header.navColorWhite {
+    .navbar {
+      .hamburger {
+        .bar {
```

Comment:
Er det noen grunn til at du lager enne selektoren så sterk? Hadde det ikke holdt med `header.navColorWhite .bar {` her?

Reply:
Da ville jeg sett på om man kunne kuttet ned på spesifisiteten i filen generelt sett. Over ser jeg `header.navColorWhite .navbar nav ul li.borderLeft` for eksempel. Den kunne blitt forenklet til `.borderLeft`